docs/design.md

```diff
@@ -411,42 +415,53 @@ is choosing a timestamp guaranteed to be greater than the latest
 timestamp of any committed transaction (in absolute time). No system can
 claim consistency and fail to read already-committed data.
 
-Accomplishing this for transactions (or just single operations)
-accessing a single node is easy. The transaction supplies 0 for
-timestamp, indicating that the node should use its current time (time
-for a node is kept using a hybrid clock which combines wall time and a
-logical time). This guarantees data already committed to that node have
-earlier timestamps.
+Time for a node is maintained by a hybrid logical clock (HLC).
+The HLC time >= wall time and is potentially updated by each write at that node.
```

Comment:
Interesting, I thought a write event observed with a timestamp < HLC doesn't affect the HLC. 

It is also my understanding that a read event doesn't update the HLC.